shared/bundle_analysis/comparison.py

```diff
@@ -1,15 +1,21 @@
+import logging
 from collections import defaultdict
 from dataclasses import dataclass
 from enum import Enum
 from functools import cached_property
 from typing import Iterator, List, MutableSet, Optional, Tuple
 
+from shared.bundle_analysis.models import MetadataKey
 from shared.bundle_analysis.report import (
     AssetReport,
     BundleAnalysisReport,
     BundleReport,
 )
 from shared.bundle_analysis.storage import BundleAnalysisReportLoader
+from shared.django_apps.core.models import Repository
+from shared.django_apps.reports.models import CommitReport
+
+log = logging.getLogger("__name__")
```

Comment:
I don't think you should put the quotes here. Just leave it `__name__` (for the logger to have the name of the file). Right now the logger would be called literally `"__name__"`, I think